packages/trust/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/trust",
-  "version": "2.1.1",
+  "version": "2.1.2-alpha.1",
```

Comment:
This same change needs to be made to the `packages/injected/package.json` to get the icon updates live for the injected module.

Reply:
I updated the injected package version to close this comment 😄